pkg/registry/chains/client/option.go

```diff
@@ -100,4 +110,5 @@ type clientOptions struct {
 	nsAdditionalFunctionality  []registry.NetworkServiceRegistryClient
 	nseAdditionalFunctionality []registry.NetworkServiceEndpointRegistryClient
 	dialOptions                []grpc.DialOption
+	dialTimeout                time.Duration
```

Comment:
Can we use a default value for the dialTimeout for the case when dialTimeout is not passed?

Reply:
We have default values ​​in the chain element. 
https://github.com/networkservicemesh/sdk/blob/f5e9a5c9a2290a69f80fd18f14b5604649574916/pkg/registry/common/dial/nse_client.go#L172
Do you think that additional ones need to be defined?

Reply:
As I can see, if `WithDialTimeout` is not passed to the chains/client then the dial timeout here  would be zero: https://github.com/networkservicemesh/sdk/blob/f5e9a5c9a2290a69f80fd18f14b5604649574916/pkg/registry/common/dial/nse_client.go#L172